tests/everflow/test_everflow_ipv6.py

```diff
@@ -0,0 +1,592 @@
+        test_packet = self._base_tcp_packet(
+            ptfadapter,
+            setup_info,
+            src_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:0008",
+            dst_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:0009",
+            sport=12000,
+            dport=443,
+            flags=0x2
+        )
+
+        self._send_and_check_mirror_packets(setup_info,
+                                            setup_mirror_session,
+                                            ptfadapter,
+                                            duthost,
+                                            test_packet)
+
+        test_packet = self._base_tcp_packet(
+            ptfadapter,
+            setup_info,
+            src_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:0009",
+            dst_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:0008",
+            sport=443,
+            dport=12000,
+            flags=0x12
+        )
+
+        self._send_and_check_mirror_packets(setup_info,
+                                            setup_mirror_session,
+                                            ptfadapter,
+                                            duthost,
+                                            test_packet)
+
+    def test_udp_application_mirroring(self, setup_info, setup_mirror_session, ptfadapter, duthost):
+        """Verify that we can match UDP traffic between a client and server application."""
+        test_packet = self._base_udp_packet(
+            ptfadapter,
+            setup_info,
+            src_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:000a",
+            dst_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:000b",
+            dscp=8,
+            sport=12001,
+            dport=514
+        )
+
+        self._send_and_check_mirror_packets(setup_info,
+                                            setup_mirror_session,
+                                            ptfadapter,
+                                            duthost,
+                                            test_packet)
+
+        test_packet = self._base_udp_packet(
+            ptfadapter,
+            setup_info,
+            src_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:000b",
+            dst_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:000a",
+            dscp=8,
+            sport=514,
+            dport=12001
+        )
+
+        self._send_and_check_mirror_packets(setup_info,
+                                            setup_mirror_session,
+                                            ptfadapter,
+                                            duthost,
+                                            test_packet)
+
+    def test_any_protocol(self, setup_info, setup_mirror_session, ptfadapter, duthost):
+        """Verify that ALL protocols are matched if no Next Header is specified."""
+        test_packet = self._base_tcp_packet(
+            ptfadapter,
+            setup_info,
+            src_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:000c",
+            dst_ip="ffbe:0225:7c6b:a982:d48b:230e:f271:000d",
```

Comment:
will it not be TCP packet even though we did not specify Next Header .
Same for UDP below

Reply:
It will be, yeah! I think the comment is misleading. We're basically testing if we have a rule that tests src ip/dst ip/src port/dst port, then we can have any Next Header we want and the rule will still match. I'll update the comment.